shell/src/main/java/tachyon/shell/TfsShell.java

```diff
@@ -101,7 +101,8 @@ public void printUsage() {
     System.out.println("Usage: java TfsShell");
     SortedSet<String> sortedCmds = new TreeSet<String>(mCommands.keySet());
     for (String cmd : sortedCmds) {
-      System.out.println("       [" + mCommands.get(cmd).getUsage() + "]");
+      System.out.format("%-60s%-95s%n", "       [" + mCommands.get(cmd).getUsage() + "]   ",
```

Comment:
I guess would be good to keep those white spaces before and after the command and usage information.